### DISCUSSION TITLE
Is it ok to use truborepo in production runtime?

### DISCUSSION THREAD

**Discussion by ctretyak** [vercel/turborepo#2098] [Help] [answered]
I mean, can I start an application with "turbo run start --parallel", that will run frontend and backend in production runtime (e.g. docker container in k8s pod) in parallel mode? And is it possible to shut down gracefully (with exit code 0)?

**Comment by nathanhammond** [comment, marked as answer]
You shouldn't do this. Turborepo is intended for build time and is not intended to coordinate runtime tasks.
